src/PrimaryAuthController.js

```diff
@@ -141,9 +142,14 @@ function (Okta, PrimaryAuthForm, SocialAuth, PrimaryAuthModel, Util, BaseLoginCo
 
       // Add SocialAuth view only when the idps are configured. If configured, 'socialAuthPositionTop'
       // will determine the order in which the social auth and primary auth are shown on the screen.
+      var addButtonSettings = this.model.settings.get('features.addButton');
+
       if (options.settings.get('socialAuthConfigured')) {
         this.add(SocialAuth, {prepend: options.settings.get('socialAuthPositionTop')});
       }
+      if (addButtonSettings && addButtonSettings.visibleWhen) {
```

Comment:
Actually, thinking about it more, we should always add the divider, and we can support an array of buttons.